### PR TITLE
Feature/lesson10

### PR DESCRIPTION
**Pull request by EvgeniyMakuta**
'check' commits are not ready for verifying

### CODE REVIEW

File: src/test/java/pages/ContextMenuPage.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import javax.swing.*;
+
+import static org.testng.Assert.fail;
+
+public class ContextMenuPage extends BasePage{
+    public static final String ENDPOINT_CONTEXT_MENU = "/context_menu";
+    public static final By BOX_LOCATOR = By.id("hot-spot");
+
+    Actions actions;
+    Alert alert;
```
Comment (vchesnakoutest) [inline src/test/java/pages/ContextMenuPage.java:16]: better to create new objects in methods where they're used and not to share them across methods

File: src/test/java/pages/TablesPage.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class TablesPage extends BasePage {
+    final String ENDPOINT = "/tables";
+    public static final By TABLE1 = By.id("table1");
+    private String cell = "";
```
Comment (vchesnakoutest) [inline src/test/java/pages/TablesPage.java:12]: this is not used, we can remove it (try not to keep extra code in your projects)

**Review by vchesnakoutest — APPROVED** [review]
make fixes and merge it

File: src/test/java/pages/ContextMenuPage.java (hunk 1 above)
Comment (EvgeniyMakuta) [reply src/test/java/pages/ContextMenuPage.java:16]: I moved Action to the used method but Alert is used in several methods therefore it's better off to keep it apart. Am I right?

File: src/test/java/pages/TablesPage.java (hunk 2 above)
Comment (EvgeniyMakuta) [reply src/test/java/pages/TablesPage.java:12]: I've just haven't finished with this class=) I send it to u by mistake

File: src/test/java/pages/ContextMenuPage.java (hunk 1 above)
Comment (vchesnakoutest) [reply src/test/java/pages/ContextMenuPage.java:16]: Not actually, that is a bad practice and also not 'thread save'

**Pull request merged**